account_move_partner_bank_by_team/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.depends("bank_partner_id", "team_id")
+    def _compute_partner_bank_id(self):
+        super()._compute_partner_bank_id()
+        for move in self:
+            if move.team_id.bank_account_id:
+                move.partner_bank_id = move.team_id.bank_account_id
+        return
```

Comment:
```suggestion
```
I think it's not necessary

Reply:
I suppose it's related to this discussion: https://github.com/OCA/pylint-odoo/issues/450

@AungKoKoLin1997 I wasn't sure if the linter still required a return. Can you confirm?

Reply:
It is still required.
```
************* Module account_move_partner_bank_by_team.models.account_move
account_move_partner_bank_by_team/models/account_move.py:11: [W8110(missing-return), AccountMove._compute_partner_bank_id] Missing `return` (`super` is used) in method _compute_partner_bank_id.

Check for Odoo modules using pylint......................................Failed
- hook id: pylint_odoo
- exit code: 4

************* Module account_move_partner_bank_by_team.models.account_move
account_move_partner_bank_by_team/models/account_move.py:11: [W8110(missing-return), AccountMove._compute_partner_bank_id] Missing `return` (`super` is used) in method _compute_partner_bank_id.
```

Reply:
Thank you for confirming it.